spec/fixtures/18TN/7818.json

```diff
@@ -1927,14 +1927,6 @@
       ],
       "original_id": 193
     },
-    {
-      "type": "dividend",
-      "entity": "GMO",
-      "entity_type": "corporation",
-      "id": 203,
-      "kind": "withhold",
-      "original_id": 194
-    },
```

Comment:
Removal of actions no longer required for 18TN and are instead handled dynamically.